lib/rqrcode.rb

```diff
@@ -1,19 +1,8 @@
-#!/usr/bin/env ruby
-
-#--
-# Copyright 2008 by Duncan Robertson (duncan@whomwah.com).
-# All rights reserved.
-
-# Permission is granted for use, copying, modification, distribution,
-# and distribution of modified versions of this work as long as the
-# above copyright notice is included.
-#++
-
-$LOAD_PATH.unshift(File.dirname(__FILE__))
-
-require "rqrcode/core_ext"
-require "rqrcode/qrcode"
-require 'rqrcode/export/png'
-require 'rqrcode/export/svg'
-require 'rqrcode/export/html'
-require 'rqrcode/export/ansi'
+# frozen_string_literal: true
+
+module RQRCode
+  require 'rqrcode_core'
+  require "rqrcode/qrcode"
```

Comment:
(minor) prefer single quotes on lines 5,6,7.

Reply:
I wonder if I plumb in `rubocop` or such like at some point?